src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Mappers.kt

```diff
@@ -172,14 +173,34 @@ fun JiraComment.toDomain(
 
 fun JiraUser.toDomain(jiraClient: JiraClient) = User(
     name, displayName,
-    ::getUserGroups.partially1(jiraClient).partially1(name)
+    ::getUserGroups.partially1(jiraClient).partially1(name),
+    ::isNewUser.partially1(jiraClient).partially1(name)
 )
 
 private fun getUserGroups(jiraClient: JiraClient, username: String) = getGroups(
     jiraClient,
     username
 ).fold({ null }, { it })
 
+private fun isNewUser(jiraClient: JiraClient, username: String): Boolean {
+    val escapedUserName = if (username.contains('\'')) """\\"$username\\"""" else "'$username'"
```

Comment:
This might break if the user name contains double quotes (or are they illegal in Jira?)

```suggestion
    val escapedUserName = if (username.contains('\'')) """\\"$username\\"""" else "'${username.replace("\"", "\\\"")}'"
```